python/assistant-stream/src/assistant_stream/modules/tool_call.py

```
@@ -48,6 +49,14 @@ def set_result(self, result: Any) -> None:
         self.loop.call_soon_threadsafe(self.queue.put_nowait, chunk)
         self.close()
 
+    def unstable_set_artifact(self, artifact: Any) -> None:
```

Ruff (0.8.2)

Dynamically typed expressions (typing.Any) are disallowed in `artifact`

(ANN401)